src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig

```diff
@@ -38,7 +38,14 @@
                 </span>
             </div>
 
-            {% if 'n/a' is not same as(collector.env) %}
+            {% if collector.mode is defined %}
```

Comment:
I suggest adding a comment that this condition is a BC layer, so that it can be cleaned when WebProfilerBundle becomes incompatible with HttpKernel <5.2